client/src/components/screens/Patient/index.jsx

```diff
@@ -112,7 +112,7 @@ class PatientScreen extends React.Component {
         title: (
           <span className="tabName">
             <MedicineBoxFilled />
-            {intl.get('screen.patient.tab.prescriptions')}
+            {`${intl.get('screen.patient.tab.prescriptions')}  (${this.props.prescriptions.length})`}
```

Comment:
make sure that you will always have prescriptions so that this.props.prescriptions.length doesn't kaboom